libs/forecastle/src/Forecastle/Services/Galley.hs

```diff
@@ -1,4 +1,4 @@
-module API.Util where
+module Forecastle.Services.Galley where
```

Comment:
This module has been moved wholesale. Probably good to come through later and if it makes sense to share all of them, and only expose the public parts of it.

For now it's mostly just generalizing the type sigs of all the helpers so they could be used in other projects.